runtime/core/event_tracer_hooks_delegate.h

```diff
@@ -46,8 +46,11 @@ namespace runtime {
  * @param[in] delegate_debug_id The id of the delegate event. If string
```

Comment:
Thanks for updating this file! In this PR please help me to keep the interface as it is rn. We can directly crash the system by applying `ET_CHECK_WITH_MSG` on the return value of `event_tracer` functions. 
The reason is this file is heavily used by our codegen and we need to update that side as well. Let me create another issue in the future to give you a better context.